restore.cmd

```diff
@@ -1,4 +1,3 @@
-yarn install
-dotnet restore Fable.proj
-dotnet restore ozmo
-dotnet restore react-todomvc
\ No newline at end of file
+call yarn install
+call dotnet restore Fable.proj
+call dotnet restore Fable.Samples.sln
```

Comment:
Oh! 😮 So we have to use the solution name explicitly, that's why I couldn't restore all the projects of the solution at once (I was just typing `dotnet restore`). Thanks for the tip!